pkg/gpu/nvidia/manager.go

```diff
@@ -185,16 +232,37 @@ func (ngm *nvidiaGPUManager) Serve(pMountPath, kEndpoint, pluginEndpoint string)
 					glog.Infoln("device-plugin registered with the kubelet")
 				}
 
-				// This is checking if the plugin socket was deleted. If so,
-				// stop the grpc server and start the whole thing again.
+				// This is checking if the plugin socket was deleted
+				// and also if there are additional GPU devices installed.
+				// If so, stop the grpc server and start the whole thing again.
+				gpuCheck := time.NewTicker(10 * time.Second)
```

Comment:
Perhaps define gpuCheckInterval and pluginSocketCheckInterval as const and use them in tests?

Reply:
Thanks, added as consts. However it looks like the plugin socket interval is currently only used here and not in the tests?